src/components/MetaData/index.tsx

```diff
@@ -461,6 +472,8 @@ MetaData.propTypes = {
       metadata_description: PropTypes.string,
       metadata_title: PropTypes.string,
     }),
+    canonical_website: PropTypes.string,
```

Comment:
`canonical_website` isn't used anywhere do we still need it defined?

Reply:
updated